### DISCUSSION TITLE
Is it possible to host on replit

### DISCUSSION THREAD

**Discussion by ThnksCJ** [Chevereto/Chevereto-Free#123] [Q&A]
Is it possible to host this on replit if so please message me because most of my website stuff is hosted on it.

**Comment by BasToTheMax** [comment]
I don't think so. Replit doesn't support MySQL connections

**Reply by rodber** [reply]
🤔 https://replit.com/talk/templates/Getting-started-with-MySql/33744

**Comment by ThnksCJ** [comment]
.
